Sources/ViewStateMachine.swift

```diff
@@ -255,3 +244,51 @@ public class ViewStateMachine {
 		}
 	}
 }
+
+// MARK: - Constraints
+
+private extension ViewStateMachine {
+	static let constraintIdentifier = "stateful-insets"
```

Comment:
Good first step, but it's recommended to only update the constraints that've actually changed. So if only top changed (and the rest didn't), update the `constant` of that constraint, and leave the rest as-is.

So there's 2 scenario's:
- If there are no constraints -> create them (like now)
- If there are constraints -> update the constants with new values

Note: I don't think re-setting the constant to the same value is considered a change, so should be ok